target/scripts/helpers/log.sh

```diff
@@ -116,4 +106,24 @@ function _log
   fi
 }
 
-function _log_with_date { _log "${1}" "$(date '+%Y-%m-%d %H:%M:%S')  ${2}" ; }
+function _log_with_date() {
+  _log "${1}" "$(date '+%Y-%m-%d %H:%M:%S')  ${2}"
+}
+
+function _get_log_level_or_default() {
+  if [[ -n ${LOG_LEVEL+set} ]]
+  then
+    printf '%s' "${LOG_LEVEL}"
+  elif [[ -e /etc/dms-settings ]]
+  then
+    local LOG_LEVEL
+    LOG_LEVEL=$(grep "^LOG_LEVEL=" /etc/dms-settings | cut -d '=' -f 2)
+    printf '%s' "${LOG_LEVEL:1:-1}"
```

Comment:
I tried to follow what `LOG_LEVEL:1:-1` is intended for since you compare to strings... I looked at the original PR [commit that introduced this](https://github.com/docker-mailserver/docker-mailserver/pull/2506/commits/f0668344379b742839ccaf64fa4f1834815164fc), but didn't see anything to explain that either.

I assume it's for triggering the `*` case?

Reply:
`grep "^LOG_LEVEL=" /etc/dms-settings | cut -d "'" -f 2` should also work

Reply:
> `grep "^LOG_LEVEL=" /etc/dms-settings | cut -d "'" -f 2` should also work

I implemented this in #2754 :)